automation/test-lock-deduplicated.sh

```diff
@@ -10,7 +10,10 @@ npm i
 if ! diff -q npm-shrinkwrap.json npm-shrinkwrap.json.old > /dev/null; then
   rm npm-shrinkwrap.json.old
   echo "** npm-shrinkwrap.json was not deduplicated or not fully committed - FAIL **";
-  echo "** Please run 'npm ci', followed by 'npm dedupe' **";
+  echo "** This can usually be fixed with: **";
+  echo "** git checkout master -- npm-shrinkwrap.json **";
+  echo "** rm -rf node_modules **";
+  echo "** npm install && npm dedupe && npm install **";
```

Comment:
The reason for advising `rm -rf node_modules && npm install` instead of `npm ci` (which would usually achieve the same) is that `package.json` may have been modified in a way that makes it incompatible with `npm-shrinkwrap.json` from the `master` branch, in which case the result is:
```
cipm can only install packages when your package.json and package-lock.json or npm-shrinkwrap.json are in sync. Please update your lock file with `npm install` before continuing.
npm ERR! 
npm ERR! 
npm ERR! Invalid: lock file's etcher-sdk@2.0.20 does not satisfy etcher-sdk@^5.2.2
```
... as kindly shared by @mbalamat 👍